kivy/core/video/video_ffpyplayer.py

```diff
@@ -218,6 +218,12 @@ def _next_frame_run(self):
         did_dispatch_eof = False
         seek_queue = self._seek_queue
 
+        # Wait for first frame to load before playing so audio doesnt get out of sync too badly
+        frame = None
+        while frame is None:
+            frame, value = ffplayer.get_frame(force_refresh=True)
+            sleep(0.1)
+
```

Comment:
I'm not sure this is ok, because this can lead to never timing out and waiting here forever, e.g. if this file happens to not actually have video, just audio. Also, this leads to discarding the first video frame.

But I'm not sure what this solves? Does the audio start playing before the video? This shouldn't happen because the player starts out paused so it shouldn't start playing audio until we unpause it, which should happen when the first frame is about ready when the metadata becomes available.